DOC: Improve the format of a math operation

star1327p opened the pull request:
<!--         ----------------------------------------------------------------
                MAKE SURE YOUR PR GETS THE ATTENTION IT DESERVES!
                ----------------------------------------------------------------

*  FORMAT IT RIGHT:
      https://www.numpy.org/devdocs/dev/development_workflow.html#writing-the-commit-message

*  IF IT'S A NEW FEATURE OR API CHANGE, TEST THE WATERS:
      https://www.numpy.org/devdocs/dev/development_workflow.html#get-the-mailing-list-s-opinion

*  HIT ALL THE GUIDELINES:
      https://numpy.org/devdocs/dev/index.html#guidelines

*  WHAT TO DO IF WE HAVEN'T GOTTEN BACK TO YOU:
      https://www.numpy.org/devdocs/dev/development_workflow.html#getting-your-pr-reviewed
-->

Improve the format of a math operation in:
https://numpy.org/devdocs/dev/internals.code-explanations.html

Improved version:
<img width="212" height="45" alt="image" src="https://github.com/user-attachments/assets/5791a00b-57c1-419f-b47c-715e34fd0216" />

Original version:
<img width="280" height="48" alt="image" src="https://github.com/user-attachments/assets/c48849ed-cab1-4733-a3f0-73c7723a8e19" />

jeanwsr committed:
ENH: MAINT: add object copy for f2py meson backend

jeanwsr committed:
TST: add test for f2py meson with object file

math-hiyoko committed:
enh: extend coverage

math-hiyoko committed:
enh: coverage

math-hiyoko committed:
fix: lint

math-hiyoko committed:
fix: change array_size

math-hiyoko committed:
fix: change length of string

math-hiyoko committed:
fix: change length of string

math-hiyoko committed:
fix: change parameter

math-hiyoko committed:
fix: change parameter

math-hiyoko committed:
fix: change parameter

math-hiyoko committed:
fix: change parameter

math-hiyoko committed:
fix: change parameter

ixgbe committed:
ENH, SIMD: Optimize the logical implementation based on Highway Wrapper

math-hiyoko committed:
fix: change parameter

math-hiyoko committed:
fix: change parameter

jorenham committed:
TYP: fix ``np.bool`` method declarations

jorenham committed:
TYP: appease ruff

star1327p committed:
DOC: Improve the format of a math operation

SanjaySakamuri committed:
Update VXE and VXE2 detection regex patterns

jorenham commented:
For me the difference looks a bit different 🤔 

<img width="1335" height="874" alt="image" src="https://github.com/user-attachments/assets/cc90cd3d-1e39-4032-8dba-e45c0e2596ae" />

jorenham committed:
TYP: fix ``np.number`` method declarations

jorenham committed:
TYP: fix ``np.integer`` method declarations

jorenham committed:
TYP: fix ``np.bool`` comparison method declarations

jorenham committed:
TYP: fix ``np.signedinteger`` method declarations

star1327p committed:
Update internals.code-explanations.rst [skip actions][skip azp][skip cirrus]

Co-authored-by: Matti Picus <matti.picus@gmail.com>

jorenham committed:
TYP: fix ``np.unsignedinteger`` method declarations

ngoldbaum committed:
MAINT: update spin to 0.14 in requirements files

kkwli committed:
TST: update test_regression::test_gph25784 (#29714)

- restrict Windows only
- use options that are supported on all compilers

bwhitt7 committed:
TST: Replace test_deprecations setup/teardown with context manager (#29692)

* WIP: Replace xunit setup with methods

* WIP: Revert "WIP: Replace xunit setup with methods"

This reverts commit 87c374fd36d668d0ff2e14c6cea9340667fcb4cd.

* TST: Change setup to context manager

* TST: Remove thread_unsafe marker

nbidler committed:
DOC: clarify numpy.asarray, numpy.asanyarray, numpy.asarray_chkfinite parameter 'order'  (#29655)

* DOC: clarified numpy.asarray parameter 'order'

Changed description to make explicit that setting 'A' requires
user to set 'copy' to 'True' to get the described behavior.
Made explicit that 'A' has two cases where it acts as
equivalent to setting 'F'. See 28247.

* DOC: clarified numpy.asarray parameter 'order'

See 28247. Changed description of 'order' to make
the behavior of 'A' more explicit and how to cause it to
preserve non-contiguous Fortran-order. Changed description
of 'C' and 'F' to clarify that output will be contiguous.

* DOC: clarified numpy.asarray() parameter 'order'

See 28247. To differentiate between 'A' and 'K', made explicit
that 'A' will not ensure the output is contiguous. 'K' has more
complex behavior related to flags and stride detection, but will
try to keep the output as close as possible to the input.

* DOC: clarify numpy.asarray parameter 'order'

See 28247. Made explicit that 'C' and 'F' can copy input to output,
'A' does not ensure contiguous output.

* DOC numpy.asarray parameter 'order' description added line break

* DOC: Additional request to change identical text blocks.

* DOC: fixed linebreak position

kkwli committed:
TST: not to include the LONGDOUBLE test on AIX

star1327p committed:
Conserve CI resources [skip actions][skip azp][skip cirrus]

dependabot[bot] committed:
MAINT: Bump github/codeql-action from 3.30.1 to 3.30.2

Bumps [github/codeql-action](https://github.com/github/codeql-action) from 3.30.1 to 3.30.2.
- [Release notes](https://github.com/github/codeql-action/releases)
- [Changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md)
- [Commits](https://github.com/github/codeql-action/compare/f1f6e5f6af878fb37288ce1c627459e94dbf7d01...d3678e237b9c32a6c9bffb3315c335f976f3549f)

---
updated-dependencies:
- dependency-name: github/codeql-action
  dependency-version: 3.30.2
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

kkwli committed:
TST: xfail test_kind::test_quad_precision on AIX/PPC (#29693)

* TST: xfail test_kind::test_quad_precision on AIX/PPC

* Change to use platform.system() instead

sandeepgupta12 committed:
CI: Add native ``ppc64le`` CI job using GitHub Actions (#29212)

mattip commented:
Something went haywire with the force-push.
